noggin_messages/__init__.py

```diff
@@ -1,9 +1,100 @@
 from fedora_messaging import message
+from fedora_messaging.schema_utils import user_avatar_url
 
 
-class MemberSponsorV1(message.Message):
+class NogginMessage(message.Message):
+    @property
+    def app_name(self):
+        """
+        Return the name of the application that generated the message.
+
+        Returns:
+            the name of the application (fas)
+        """
+        return "fas"
```

Comment:
This is supposed to be human-readable, so you can set "FAS" here, or "Account System"